src/components/typography/Title.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { StyleProp, Text, TextStyle } from "react-native";
+import { useTheme } from "smartway-react-native-ui";
+import type { TextProps } from "./TextProps";
+
+export const Title = ({ children, style }: TextProps) => {
+    const theme = useTheme();
+    const titleStyle: StyleProp<TextStyle> = {
+        color: theme.sw.colors.neutral[800],
```

Comment:
dans le DS c'est GreyNeutral[800] 😂

Reply:
Monsieur pinaille !

Je vais voir avec Améline: dans la doc [Material UI il parle juste de neutral color](https://m3.material.io/styles/color/the-color-system/key-colors-tones) et pas de grey neutral.

Reply:
Et grey quand t'as du noir et du blanc dedans c'est chelou ^^

Reply:
je vois, du coup c'est peut etre le DS qui devrait etre maj dans ce cas ^^

Reply:
C'est validé avec Améline, c'est bien "neutral" !